src/olympia/amo/fixtures/base/users.json

```diff
@@ -215,7 +215,7 @@
         "pk": 50004,
         "model": "access.group",
         "fields": {
-            "rules": "Addons:Review,Addons:ReviewUnlisted,Reviews:Edit,ReviewerAdminTools:View",
+            "rules": "Addons:Review,Addons:PostReview,Addons:ReviewUnlisted,Reviews:Edit,ReviewerAdminTools:View",
```

Comment:
this will probably need adding to the Staff group too.  (On Marketplace I just changed the staff rule to be `Addons:*` iirc)

Though the staff group is only defined in `access/fixtures/initial.json` so I dunno.